tests/phpunit/integration/data/story_post_content_filtered.json

```diff
@@ -1,153 +1,153 @@
-          "width": 1,
-          "height": 1,
-          "mask": {
-            "type": "rectangle"
-          },
-          "isBackground": true,
-          "isDefaultBackground": true,
-          "type": "shape",
-          "id": "d58c2be2-f8a0-4e00-b21a-ddd32d1ac04b"
-        },
-        {
-          "opacity": 100,
-          "flip": {
-            "vertical": false,
-            "horizontal": false
-          },
-          "rotationAngle": 0,
-          "lockAspectRatio": true,
-          "backgroundTextMode": "NONE",
-          "font": {
-            "family": "Roboto",
-            "weights": [
-              100,
-              300,
-              400,
-              500,
-              700,
-              900
-            ],
-            "styles": [
-              "italic",
-              "regular"
-            ],
-            "variants": [
-              [
-                0,
-                100
-              ],
-              [
-                1,
-                100
-              ],
-              [
-                0,
-                300
-              ],
-              [
-                1,
-                300
-              ],
-              [
-                0,
-                400
-              ],
-              [
-                1,
-                400
-              ],
-              [
-                0,
-                500
-              ],
-              [
-                1,
-                500
-              ],
-              [
-                0,
-                700
-              ],
-              [
-                1,
-                700
-              ],
-              [
-                0,
-                900
-              ],
-              [
-                1,
-                900
-              ]
-            ],
-            "fallbacks": [
-              "Helvetica Neue",
-              "Helvetica",
-              "sans-serif"
-            ],
-            "service": "fonts.google.com"
-          },
-          "fontSize": 14,
-          "backgroundColor": {
-            "color": {
-              "r": 196,
-              "g": 196,
-              "b": 196
-            }
-          },
-          "lineHeight": 1.3,
-          "textAlign": "initial",
-          "padding": {
-            "vertical": 0,
-            "horizontal": 0,
-            "locked": true
-          },
-          "type": "text",
-          "id": "e781bc30-4ebe-416f-bf1d-b509dc3ef6ce",
-          "content": "<span style=\"font-weight: 700\">Fill in some text<\/span>",
-          "x": 73,
-          "y": 138,
-          "width": 160,
-          "height": 18,
-          "scale": 100,
-          "focalX": 50,
-          "focalY": 50
-        }
-      ],
-      "backgroundColor": {
-        "color": {
-          "r": 255,
-          "g": 255,
-          "b": 255
-        }
-      },
-      "type": "page",
-      "id": "a3f18c09-3f10-43c4-9446-794f0541ea1e"
-    }
-  ],
-  "autoAdvance": true,
-  "defaultPageDuration": 7
+	"version": 21,
```

Comment:
You might want to tell your IDE to not accidentally change indentation of JSON files like this :-)